src/views/Auth/ResetPassword.jsx

```diff
@@ -0,0 +1,163 @@
+    })
+
+    console.log(this.state.resetToken)
+  }
+
+  handleEmailChange(event) {
+    this.setState({
+      email: event.target.value
+    })
+  }
+
+  handleNewPasswordChange(event) {
+    this.setState({
+      newPassword: event.target.value
+    })
+  }
+
+  handleConfirmNewPasswordChange(event) {
+    this.setState({
+      confirmNewPassword: event.target.value
+    })
+  }
+
+  resetStateStatusFlags() {
+    this.setState({
+      passwordMismatch: false,
+      passwordsEmpty: false,
+      resetSuccess: false,
+      resetError: false
+    })
+  }
+
+
+  //TODO: Temporary behavior to always reset to login screen
+  submitForm(event) {
+    console.log("submit reset password form")
+    event.preventDefault()
+    this.resetStateStatusFlags()
+
+    if (!this.state.newPassword && !this.state.confirmNewPassword) {
+      this.setState({
+        passwordsEmpty: true
+      })
+      return
+    } else if (this.state.newPassword !== this.state.confirmNewPassword ||
+        !this.state.newPassword ||
+        !this.state.confirmNewPassword) {
+      this.setState({
+        passwordMismatch: true,
+      })
+      return
+    }
+
+    var client = new APIClient();
+    client.resetPassword(this.state.email, this.state.newPassword, this.state.resetToken).then(() =>{
+      this.setState({
+        resetSuccess: true
+      })
+    }).catch(() => {
+      this.setState({
+        resetError: true
+      })
+    })
+  }
+
+  render() {
+    return (
+        <>
+          <div style={{marginTop: "100px"}}>
+            <Container>
+              <Row>
+                <Col sm={{size: 10, offset: 1}} md={{ size: 4, offset: 4 }}>
+                  <Card data="blue"  style={{background: "linear-gradient(0deg,#3358f4,#1d8cf8)"}}>
+                    <CardHeader>
+                      <h2 style={{textAlign: "center", color: "white"}}>Reset Password</h2>
+                    </CardHeader>
+                    <CardBody>
+                      <Form onSubmit={ (e) => this.submitForm(e) }>
+                        {this.state.resetSuccess && <p style={{color: "white"}}>Reset successful, please return to login page!</p>}
+                        {this.state.resetError && <p style={{color: "white"}}>Reset failed, check your email address and try again!</p>}
+                        <FormGroup>
+                          <Label for="exampleEmail" style={{color: "white"}}>Email</Label>
+                          <Input type="email"
+                                 name="email"
+                                 id="email"
+                                 placeholder="email"
+                                 style={{color: "white"}}
+                                 value={this.state.email}
+                                 onChange={(e) => this.handleEmailChange(e)}/>
+                        </FormGroup>
+                        {this.state.passwordsEmpty && <p style={{color: "white"}}>Passwords are empty</p>}
```

Comment:
We should use reactstrap alerts for this.

Reply:
Agreed